ng.html

```diff
@@ -20,12 +20,50 @@ <h2> <span class="add-update">Add</span> Gloss </h2>
         NB: A Gloss is an important thing. Please describe it as well as possible and
         avoid creating a new one unless it does not already exist.
     </p>
+
+
+    <gog-reference-widget></gog-reference-widget>   
+
+    <div id="bibliographicCitationModal" class="bib-citation-modal">
```

Comment:
Is there a reason why this couldn't be a custom HTML element like we do with the other entities (/scripts/witnessModal.js, /scripts/glossModal.js)?  That may be out of scope now.  Just wondering.

Reply:
I made it a widget because I thought it was very similar to tag/theme widget, so I just made it a widget as well. It would probably be easier to follow if it was just an html element.

Reply:
I was planning to, but I think it was a lot faster to get the add references part working by just leaving the modal as is. I could plan to turn it into a custom HTML element when working on issue 143 since that has to deal with the modal actually adding references to a gloss.